src/hamiltonian.F90

```diff
@@ -657,6 +657,23 @@ subroutine hamiltonian_write_rmn()
     end do
     
     close(file_unit)
+!  
+!   RM and SP - write to file vectors b and their weight wb for each k-point 
+!   This is used by EPW to compute the velocity. 
+!   Note that you need to put in your Wannier input:
+!     write_hr        = .true.
```

Comment:
Instead of requiring all three flags to be set, could you put the code in a different function, and in `plot_main` (plot.F90) you do, toward the end, below
```
if(wannier_plot) call plot_wannier
```
something like
```
if(write_bvec) call plot_bvec
```

Note that the file will be changed when merging #177, this automatically also remove the need of `write_hr` (so you might want to work on top of those changes - in this case, add also the `.or. write_bvec` at the top, where the header is printed).